.travis.yml

```diff
@@ -7,14 +7,11 @@ node_js:
   - "6"
   - "7"
 
-matrix:
-  allow_failures:
-    - node_js: "0.12"
-    - node_js: "node"
- 
 branches:
-  except:
-    - canary
+  only:
+    - master
+    - beta
+    - release
```

Comment:
here is what you need for greenkeeper https://github.com/kellyselden/package-hint-historic-resolver/blob/master/.travis.yml#L11